cachito/workers/pkg_managers/gomod.py

```diff
@@ -308,13 +308,18 @@ def resolve_gomod(app_source_path, request, dep_replacements=None, git_dir_path=
 
             pkg_level_deps = []
             for dep_name in package_info[pkg_name].get("Deps", []):
+                # this check still passes with standard lib
                 dep_info = package_info.get(dep_name)
                 if dep_info is None:  # dependency is from the standard library
                     continue
```

Comment:
Oh by this comment I actually meant that even before any of our changes this `if` doesn't do much. Will `dep_info` ever be null given it's initialized `{}`? Or even if we accounted for that,  it'd still rely on the stdlib having no `Deps`. 

So we're pretty sure this if statement can be removed entirely, as even if it has no deps, but is a stdlib we'll still need to record it.

Reply:
Before your changes, the `_load_list_deps()` function would exclude stdlib packages, so the check was relevant. Now not anymore, :+1:  to remove it

Previously, the structure of `package_info` was something like this:
```python
{
  "github.com/this-repo/this-package": {
    "Module": {"Path": "github.com/this-repo/this-package", ...},
    "Deps": ["github.com/other-repo/some-dep", "fmt", ...]
  },
  "github.com/other-repo/some-dep": {
    "Module": {...},
    "Deps": [...]
  },
  # fmt: not present because it's stdlib
}
```
For every package and dep it lists their module and list of dependencies, but stdlib deps are not present. After the changes, stdlib deps will be present too
```python
{
  ...,
  "fmt": {"Standard": True, "Deps": [...]}
}
```